cluster/gce/gci/configure-helper.sh

```diff
@@ -1453,6 +1453,14 @@ function create-master-etcd-apiserver-auth {
    fi
 }
 
+function docker_installed {
+    if [[ $(systemctl list-units --all -t service --full --no-legend "docker.service" | cut -f1 -d' ') == "docker.service" ]]; then
```

Comment:
Is this just checking if the service exists? Would it work to just cat the service file and check the return code which might be simpler?

```
gke-test-cs-18-default-pool-104faf53-733p ~ # systemctl cat docker.service &> /dev/null ; echo $?
0
gke-test-cs-18-default-pool-104faf53-733p ~ # systemctl cat dockerfoo.service &> /dev/null ; echo $?
1
```

Reply:
good idea. Applied